docs/release-notes/RELEASE-NOTES-0.9.0.md

```diff
@@ -6,45 +6,51 @@ It:
 * Does that TODOFILLMEIN. This impacts the persisted DB;
 * Does that TODOFILLMEIN.
 
-This release introduces backward incompatible changes in the chain format.
-After upgrading your node, you will not have your previous balance (even if you keep your key pair).
+This release introduces backward incompatible changes in the chain format:
+* After upgrading your node, you will not have your previous balance (even if you keep your key pair);
+* Please ensure that you do not reuse a persisted blockchain produced by the previous releases "v0.8.x".
 
 Please join the testnet by following the instructions below, and let us know if you have any problems by [opening a ticket](https://github.com/aeternity/epoch/issues).
 Troubleshooting of common issues is documented [in the wiki](https://github.com/aeternity/epoch/wiki/Troubleshooting).
 
 The instructions below describe:
 * [How to retrieve the released software for running a node](#retrieve-the-software-for-running-a-node);
+* [How to install a node](#install-node);
 * [How to join the testnet](#join-the-testnet).
 
 ## Retrieve the software for running a node
 
-Download the [release binary](https://github.com/aeternity/epoch/releases/tag/v0.9.0) corresponding to your platform, e.g. `epoch-0.9.0-osx-10.12.6.tar.gz`; you would normally find the downloaded package in `~/Downloads` on macOS.
+You can run a node by using either:
+* The published [release binary](https://github.com/aeternity/epoch/releases/tag/v0.9.0) corresponding to your platform; or
+* The published [Docker image `aetrnty/epoch`](https://github.com/aeternity/epoch/blob/v0.9.0/docs/docker.md).
 
-The binaries are tested on the following platforms:
+The release binaries are tested on the following platforms:
```

Comment:
I think this should be moved to installation.md, mentioning the releases are published on GitHub.

Reply:
Agreed. Addressed in 06e9974b